proposals/libocispec.md

```diff
@@ -0,0 +1,37 @@
+# OCI libocispec project proposal
+
+## Abstract
+Provide a C library for parsing and handle OCI runtime spec file and OCI image spec files.
+
+## Proposal
+With repositories under the http://github.com/opencontainers organization:
+
+Rename (transfer) https://github.com/giuseppe/libocispec would become https://github.com/opencontainers/libocispec.
+
+### Initial Maintainers
+Initial maintainers of the libocispec project:
+* Giuseppe Scrivano <gscrivan@redhat.com> (@gscrivano)
+* Qiang Huang <h.huangqiang@huawei.com>
+* Daniel J Walsh <dwalsh@redhat.com> (@rhatdan)
+
+### Code of Conduct
+This project would incorporate (by reference) the OCI Code of Conduct.
+
+### Governance and Releases
+This project would incorporate the Governance and Releases processes from the OCI project template: https://github.com/opencontainers/project-template.
+
+### Project Communications
+Both of the proposed projects would continue to use existing channels in use by the OCI developer community for communication including:
+* GitHub for issues and pull requests
+* The dev@opencontainers.org email list
+* The weekly OCI developer community conference call
+* The #OpenContainers IRC channel
+
+### Versioning / Roadmap
+It aims to be a C library to be statically linked in the projects that need it.
+
+## Frequenty Asked Questions (FAQ)
+
+**Q: Does this change the OCI Charter or Scope Table?**
+
+A: No.  Nothing in this proposal is intended to amend the OCI Charter (https://www.opencontainers.org/about/governance) or OCI Scope Table (https://www.opencontainers.org/about/oci-scope-table).
```

Comment:
Really?  There's nothing in the scope table about this.  I think we should ask for an entry covering this project.  Although previous proposals that should have impacted the scope table have had their associated scope-table updates [punted to post-proposal][1], so maybe we want to go that way again.

[1]: https://github.com/opencontainers/tob/pull/22#discussion_r93338910